interface/orders/list_reports.php

```diff
@@ -161,248 +178,308 @@ function openPatient(pid) {
-                if ($errmsg) {
-                    echo "<span class='text-danger'>" . text($errmsg) . "</span><br />\n";
-                }
+    if (!empty($s)) {
+        if ($matchreqs || $errors) {
+            $orphan_orders = true;
+            echo "<h4  class='bg-success text-white mb-0'>";
+            echo xlt('Incoming results requiring attention:');
+            echo "</h4>\n";
+        }
 
-                $form_from_date = empty($_POST['form_from_date']) ? '' : trim($_POST['form_from_date']);
-                $form_to_date = empty($_POST['form_to_date']) ? '' : trim($_POST['form_to_date']);
-                // if (empty($form_to_date)) $form_to_date = $form_from_date;
+        echo "<div class='table-responsive'><table class='table table-sm mb-0'>\n";
+        echo "<thead>\n";
+        echo " <tr class='head'>\n";
+        echo "  <th>" . xlt('Delete') . "</th>\n";
+        echo "  <th>" . xlt('Processing Lab Name/Internal Lab Id/Results File Name') . "</th>\n";
+        echo "  <th>" . xlt('Message') . "</th>\n";
+        echo "  <th>" . xlt('Match') . "</th>\n";
+        echo " </tr></thead><tbody id='wait'>\n";
+        echo $s;
+        echo "</tbody></table>\n";
+
+        if ($matchreqs || $errors) { ?>
+            <div class="" data-toggle="collapse" data-target="#help">
+                <i class="fa fa-plus"></i><span id="wait"> <strong>Help</strong></span>
+            </div>
+            <div class="collapse bg-warning" id="help">
+                <?php if ($matchreqs) { ?>
+                <p>
+                    Click the returned patient line item to pull up the patient matching dialog to verify if a match is indicated or select to create a new patient. The match column shows the selected patient ID, or if 0, will create a new patient. Simply clicking Resolve Orphans again without selecting a patient will automatically create a new patient based on the orders patient information. After verifying patients, click Resolve Orphans one last time to complete transactions. A reminder message will be sent to the provider associated with order.
+                </p>
+                <?php } else { ?>
+                <p>
+                    Checkboxes indicate if you want to reject and delete the HL7 file. Clicking the Resolve Orphans button will determine if the patient associated with the manual orders exists and will then create a new encounter if one doesn't exist within 30 days of the orphaned orders order date, then will attach the new order to it. If the patient does not exist and there could be a potential match with existing patients then, on return, you will be given an opportunity to match or create a new patient to assign the orphaned order. Otherwise, if there is not any ambiguity that the concerning patient of the order does not currently exist, then the patient will be automatically created and logged to messages and results returned below for review.
```

Comment:
consider translating these if feel need to support more than english on this stuff.